![Kainth_Headshot]
<img align="left" width="400" src="[https://github.com/kainth-amoldeep/kainth-amoldeep.github.io/assets/66103719/7582cb33-4735-45f9-9034-09a624cb1c91]">
# AMOLDEEP KAINTH
920 E 58th Street | Chicago, IL 60637 | 605-786-5657 | akainth@uchicago.edu

## EDUCATION

**PhD	LSU-Health | Shreveport, LA, USA | Department of Biochemistry and Molecular Biology	May 2019**
	<br/> Ike Muslow Pre-doctoral Fellowship

**MS	Panjab University | Chandigarh, India | Department of Biotechnology	May 2011**
	<br/> Council of Scientific and Industrial Research-Junior Research Fellowship

**BS	Panjab University | Chandigarh, India | Department of Biotechnology	May 2009**
	<br/> Top ranker in the National Entrance Examination	

## RESEARCH AND ANALYSIS EXPERIENCE	

**Dept. of Molecular Genetics and Cell Biology, University of Chicago | Chicago, IL	Nov. 2019 – Present**
<br/> *Post-doctoral Scholar: RNA-chromatin interactions in chromatin architecture and gene expression*
<br/> •	Designed, prepared libraries, analyzed and troubleshoot NGS experiments such as RNA-seq, ChIP-seq, Oxford Nanopore long-read sequencing
<br/> •	Conceived, developed, benchmarked, and implemented computational tools for integration of short- and long-read seq for transcript assembly and gene expression analysis
<br/> •	Developed computational pipelines for objective-oriented analysis, visualization, and biological interpretation of publicly available chromatin NGS datasets
<br/> •	Computational characterization of cancer cells based on lncRNA transcriptomics
<br/> •	Mentored graduate students in computational and molecular biology techniques
<br/> •	Organize and lead computational workshops for graduate students
<br/> •	Optimized processes for analytical scale heterologous expression and purification of protein and RNA
<br/> •	Developed and performed assays for analyzing protein-protein and protein-RNA interactions
<br/>
<br/>**Dept. of Biochemistry and Molecular Biology, LSU-Health | Shreveport, LA	Jan. 2014 – Oct. 2019**
<br/> *Graduate Research Assistant (Ph.D.): Spatial restructuring of stress-responsive genes*
<br/> •	Developed and performed molecular and cell biology assays to study spatial restructuring of genome
<br/> •	Integrate molecular and computational findings to test stress-induced changes in chromatin architecture
<br/> •	Mentored graduate students and high school teachers in molecular and computational biology methods
<br/> •	Co-guest editor of special volume of Methods journal on “Methods for mapping 3D genomic architecture”
<br/> •	Presented and discussed scientific findings in more than five international meetings
<br/>
<br/>**Dept. of Chemical and Biological Engineering, SDSMT | Rapid City, South Dakota	Sep. 2011 – Dec. 2013**
<br/> *Research Assistant: Structure-function relationship of thermo-stable enzymes*  
<br/> •	Computational analyses of the structural basis of thermal stability of commercially relevant enzymes
<br/> •	Comparative genomics for evolution of thermophilic bacteria
<br/> •	Taught a module in Basics of Bioinformatics and Thermodynamics in Microbiology courses to undergraduate classes
<br/>
## SKILLS
<br/>
**Computational Biology:** NGS data analysis (ChIP-seq, CUT&RUN, ATAC-seq, RNA-seq, Nanopore long-read seq, PRO-seq, Hi-C, Micro-C); Protein analysis (PyMol, Rosetta, AlphaFold); Image analysis (ImageJ, MATLAB); Coding (Python, Bash); Biostats (R, SPSS, MS Excel)
<br/> **Molecular Biology:** 3C; NGS for genomics, epigenomics and transcriptomics; Nanopore Long-read seq; RT-qPCR; Tissue culture and genetic manipulations of cells; Immuno- and epi- fluorescence confocal and super-resolution microscopy
<br/> **Biochemistry:** Heterologous expression of proteins; Affinity, ion-exchange, size exclusion chromatography etc. using AKTA systems; Large scale synthesis and purification of RNA; Characterization by activity and binding assays
<br/>
## PUBLICATIONS
<br/> •	*Kainth AS, *Haddad GA, Hall JH, Ruthenburg AJ (2023) bioarxiv https://doi.org/10.1101/2022.12.13.520317
<br/> •	Chowdhary S, Kainth AS, Paracha S, Gross DS, Pincus D (2022) Molecular Cell, 82, 4386-4399
<br/> •	*Kainth AS, *Chowdhary S, Pincus D, Gross DS (2021) Trends in Cell Biology. doi.org/10.1016/j.tcb.2021.04.004
<br/> •	Chowdhary S, Kainth AS, Gross DS (2020) Methods 170: 4-16
<br/> •	Kainth AS, Meduri R, Pandit V, Rubio LS, Gross DS (2019) EMBO J: e104077
<br/> •	*Chowdhary S, *Kainth AS, Pincus D, Gross DS (2019) Cell Reports 26: 18-28 e5
<br/> •	Chowdhary S, Kainth AS, Gross DS (2017) Molecular Cellular Biology 37
<br/> •	Anandhakumar J, Moustafa YW, Chowdhary S, Kainth AS, Gross DS (2016) Molecular Cellular Biology 36: 1943-60
<br/> •	Gross DS, Chowdhary S, Anandhakumar J, Kainth AS (2015) Current Biology 25: R1158-63
<br/> •	Bhalla A, Kainth AS, Sani RK (2013) Genome Announcement 
<br/> * Equal contribution
